lib/UserSettings.php

```diff
@@ -154,9 +154,14 @@ protected function getDefaultFromSetting($method, $type) {
 	 */
 	public function getNotificationTypes() {
 		$settings = $this->manager->getSettings();
-		return array_map(function (ActivitySettings $setting) {
+		
+		$return = array_map(function (ActivitySettings $setting) {
 			return $setting->getIdentifier();
 		}, $settings);
+		if (array_search('file_changed', $return)) {
```

Comment:
```suggestion
		if (array_search('file_changed', $return) !== false) {
```

otherwise it doesn't work if `file_changed` is in place `0`